<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view</class>
 <widget class="QMainWindow" name="view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>view</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Interpolation">
     <attribute name="title">
      <string>Interpolation</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Interpolation parametrs:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Approximation">
     <attribute name="title">
      <string>Approximation</string>
     </attribute>
     <widget class="QSpinBox" name="number_of_days">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="degree_of_polynom">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>degree_of_polynom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>number_of_days</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Approximation parametrs:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="s21::PaintScene" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="load_data">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>541</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_scene">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>224</y>
      <width>147</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>224</width>
      <height>173</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>224</y>
      <width>146</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_button">
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_button">
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>set</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_box_data">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>353</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Set Load</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_box_time">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>383</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Time range set and correct</string>
    </property>
   </widget>
   <widget class="QWidget" name="blocj_for_check_box" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="get_price">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="print_graph">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Print graph</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="number_of_points">
    <property name="geometry">
     <rect>
      <x>703</x>
      <y>76</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>76</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>number_of_points</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_price">
    <property name="geometry">
     <rect>
      <x>692</x>
      <y>167</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_price_name">
    <property name="geometry">
     <rect>
      <x>583</x>
      <y>167</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>565</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>565</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>s21::PaintScene</class>
   <extends>QWidget</extends>
   <header>view/paint_scene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
